### PR TITLE
Feature/#294 notifications

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/notification/views.py [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+class NotificationAPIView(RetrieveUpdateAPIView):
+    permission_classes = (IsAuthenticated, IsNotifiedOwner)
+    serializer_class = NotificationSerializer
+    queryset = Notification.objects.all()
+
+    def get_object(self):
+        obj = get_object_or_404(self.get_queryset(), pk=self.kwargs["pk"])
+        self.check_object_permissions(self.request, obj)
+        notified = obj.notified.filter(recipient=self.request.user).get()  # one notified per user
+        if notified.did_read is False:
```
Comment (elyk) [inline backend/unpp_api/apps/notification/views.py]: Why are we updating in `get_object`?

Comment (lkolacz) [reply backend/unpp_api/apps/notification/views.py]: I move this logic to `retrieve`

Comment (elyk) [reply backend/unpp_api/apps/notification/views.py]: We should update this on PATCH. GET request doesn't make sense to update a resource.

Comment (lkolacz) [reply backend/unpp_api/apps/notification/views.py]: @elyk only owner is able to read the notification via notified relation ... IMO that have seans.

Comment (lkolacz) [reply backend/unpp_api/apps/notification/views.py]: @elyk done

File: backend/unpp_api/apps/common/middleware.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 from django.utils.functional import SimpleLazyObject
-from django.conf import settings
```
Comment (elyk) [inline backend/unpp_api/apps/common/middleware.py]: Hm why did we remove?

**Pull request merged**